src/components/molecules/sidebar-company-logo/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const SidebarCompanyLogo = ({ imageUrl, StoreName }) => {
+  return (
+    <div className='flex px-2 mt-12 mb-3.5 items-center'>
```

Comment:
Remove margin-top from company logo. The general layout will take care of the positioning